app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -315,6 +315,7 @@ def pod_definition
               "--scan-type=openscap",
               "--serve=0.0.0.0:#{options[:pod_port]}"
             ],
+            :imagePullPolicy => "Always",
```

Comment:
@enoodle I would suggest moving ```:imagePullPolicy``` after ```:image```.
That's how it is in the example you kindly [shared](http://kubernetes.io/docs/user-guide/images/) and it makes sense

Maybe add the link to the pr first message?